transcribed/page0100.txt

```diff
@@ -9,15 +9,15 @@
 009     100 023         JRST POPJ1      ;HAPPY
 010     118 037         SKIPE FLSINS(U)
 011     010 017         JUMPN U,PCLSR1  ;BACK UP TO UUO
-012     021 002 PCLSR4: MOVE T,[JRST ONEFLS
```

Comment:
The comment should move one character to the left.

Reply:
Agreed, thanks for fixing my mistake! :)